server/plugins/api/Twitter.ts

```diff
@@ -1,22 +1,71 @@
 import { APIReturn } from "./APIReturn";
 import { GenericNotifier } from "./GenericNotifier";
 
+import * as TwitterAPI from "twit";
+
 export default class Twitter implements GenericNotifier<{}> {
 
 	public async setup(): Promise<void> {
-		return;
+		const consumerKey = process.env.TWITTER_CONSUMER_KEY;
```

Comment:
You should save these as instance variables of your Twitter class so you don't have to fetch them again. (But in that case you should probably move this code into the constructor of this class)

Reply:
Here's a snippet:
```typescript
class Twitter {
    private consumerKey: string;
    private consumerSecret: string;
    private accessToken: string;
    private accessTokenSecret: string;

    constructor() {
        const consumerKey = process.env.TWITTER_CONSUMER_KEY;
        const consumerSecret = process.env.TWITTER_CONSUMER_SECRET;
        const accessToken = process.env.TWITTER_HACKGT_ACCESS_TOKEN;
        const accessTokenSecret = process.env.TWITTER_HACKGT_ACCESS_TOKEN_SECRET;

        if (!consumerKey) {
            console.error("Missing Twitter consumer_key env var");
        }
        if (!consumerSecret) {
            console.error("Missing Twitter consumer_secret env var");
        }
        if (!accessToken) {
            console.error("Missing Twitter access_token env var");
        }
        if (!accessTokenSecret) {
            console.error("Missing Twitter access_token_secret env var");
        }

        if (!consumerKey || !consumerSecret || !accessToken || !accessTokenSecret) {
            throw new Error("Twitter env vars missing, aborting...");
        }

        this.consumerKey = consumerKey;
        this.consumerSecret = consumerSecret;
        this.accessToken = accessToken;
        this.accessTokenSecret = accessTokenSecret;
    }
}
```